pkg/api/frontend_logging.go

```diff
@@ -0,0 +1,88 @@
+package api
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/inconshreveable/log15"
+)
+
+var frontendLogger = log.New("frontend")
+
+type FrontendSentryExceptionValue struct {
```

Comment:
There's no need to export this:
```suggestion
type frontendSentryExceptionValue struct {
```